### PR TITLE
Add operator contains.

### PR DESCRIPTION
**Pull request by CPWstatic**

### CODE REVIEW

File: src/graph/test/GoTest.cpp [hunk 1]
```diff
@@ -2205,6 +2255,101 @@ TEST_P(GoTest, DuplicateColumnName) {
...
+        std::vector<std::tuple<std::string, int64_t, int64_t, std::string>> expected = {
+            {player.name(), 2003, 2005, "Hawks"},
+        };
+        ASSERT_TRUE(verifyResult(resp, expected));
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        auto &player = players_["Boris Diaw"];
+        auto *fmt = "GO FROM %ld OVER serve "
+                    "WHERE (string)serve.start_year CONTAINS \"05\" "
```
Comment (laura-ding) [inline src/graph/test/GoTest.cpp:2283]: add `upper` or `lower` function, and Chinese？

File: src/common/filter/Expressions.h [hunk 2]
```diff
@@ -313,6 +313,10 @@ class Expression {
         return std::abs(left - right) < EPSILON;
     }
 
+    static bool contains(const std::string& left, const std::string& right) {
```
Comment (wadeliuyi) [inline src/common/filter/Expressions.h:316]: is there any case that right is empty string?

File: src/graph/test/GoTest.cpp (hunk 1 above)
Comment (CPWstatic) [reply src/graph/test/GoTest.cpp:2283]: OK.

File: src/common/filter/Expressions.h (hunk 2 above)
Comment (CPWstatic) [reply src/common/filter/Expressions.h:316]: It's possible.

**Comment by whitewum** [comment]
@Amber1990Zhang

**Comment by Amber1990Zhang** [comment]
doc done.

**Pull request merged**